### ISSUE TITLE
Multiple models in publication

### ISSUE THREAD

**Issue by yannikschaelte** [LoosC/Benchmark-Models#70]
@LeonardSchmiester @PhilippStaedter please add details.

We should discuss what to do when we have multiple models in one publication. Say all use the same data and experimental conditions. So the models describe the same stuff, but maybe in more or less detail or alternative networks.

One could put all such models in the same folder and just enumerate / make the names unique. Alternative suggestions?

**Comment by yannikschaelte** [comment]
see also #43

**Comment by LeonardSchmiester** [comment]
In these cases parameters are seperated individually for every model right? But we also have cases, with different models but all in one parameter estimation (e.g. Becker)?

Maybe use subfolders for individual optimizations within the same publication?

**Comment by yannikschaelte** [comment]
good point. I guess in this case some of the parameters coincide, while others are model specific. thus one could do subfolders, or just separate folders on the global level, à la Kouril2021Model1, 2.

**Comment by LeonardSchmiester** [comment]
In favor of subfolders as it might get crowded for publications with a lot of models. Might also be easier, if you want to specify model selection.
